### DISCUSSION TITLE
TLS安装失败，请检查acme日志,怎么解决啊

### DISCUSSION THREAD

**Discussion by cs1580** [mack-a/v2ray-agent#686] [Q&A]
---> 生成证书中
[Thu 01 Jun 2023 09:14:41 PM EDT] LISTEN    0         128                0.0.0.0:80               0.0.0.0:*        users:(("nginx",pid=8785,fd=6),("nginx",pid=8784,fd=6),("nginx",pid=8783,fd=6))
[Thu 01 Jun 2023 09:14:41 PM EDT] tcp port 80 is already used by (("nginx",pid=8785,fd=6),("nginx",pid=8784,fd=6),("nginx",pid=8783,fd=6))
[Thu 01 Jun 2023 09:14:41 PM EDT] Please stop it first
[Thu 01 Jun 2023 09:14:41 PM EDT] _on_before_issue.
[Thu 01 Jun 2023 09:14:42 PM EDT] Using CA: https://acme-v02.api.letsencrypt.org/directory
[Thu 01 Jun 2023 09:14:42 PM EDT] Standalone mode.
[Thu 01 Jun 2023 09:14:42 PM EDT] LISTEN    0         128                0.0.0.0:80               0.0.0.0:*        users:(("nginx",pid=8785,fd=6),("nginx",pid=8784,fd=6),("nginx",pid=8783,fd=6))
[Thu 01 Jun 2023 09:14:42 PM EDT] tcp port 80 is already used by (("nginx",pid=8785,fd=6),("nginx",pid=8784,fd=6),("nginx",pid=8783,fd=6))
[Thu 01 Jun 2023 09:14:42 PM EDT] Please stop it first
[Thu 01 Jun 2023 09:14:42 PM EDT] _on_before_issue.
 ---> TLS安装失败，请检查acme日志
Debian 10也是这样
难受，DNS 1.1.1.1放在首位也没用,

**Comment by mack-a** [comment]
nginx手动关闭一下或者直接dd新系统重新安装
